crow/setup3.py

```diff
@@ -18,15 +18,15 @@
 except:
   eigen_flags = ""
 include_dirs=['include/distributions','include/utilities','contrib/include']
-if eigen_flags.startswith("-I"):
+if eigen_flags.startswith(b"-I"):
```

Comment:
The next line would probably fail if the if statement was true.  It probably should assume that eigen_flags is ascii and decode the bytes to unicode.